Formula/r/ruptime.rb

```diff
@@ -0,0 +1,31 @@
+class Ruptime < Formula
+  desc "poor man’s ruptime"
+  homepage "https://wedonthaveaprivacyproblem.com"
+  url "https://github.com/alexmyczko/ruptime/archive/refs/tags/1.8.tar.gz"
+  sha256 "c7115c34e11d79d1d0e966516818f3e5f7a0417aadb6cdddfb415cb103f688eb"
+  license "AGPL-3.0-or-later"
+
+  depends_on "xz"
+  depends_on "memtester"
+
+  def install
+    bin.install "ruptime"
+    #bin.install "ruptimed"
+    bin.install_symlink "ruptime" => "rsw"
+    bin.install_symlink "ruptime" => "rhw"
+    bin.install_symlink "ruptime" => "runame"
+    bin.install_symlink "ruptime" => "rbench"
+    bin.install_symlink "ruptime" => "rboot"
+    bin.install_symlink "ruptime" => "rdisk"
+    bin.install_symlink "ruptime" => "rload"
+    bin.install_symlink "ruptime" => "rnet"
+    bin.install_symlink "ruptime" => "rac"
+    bin.install_symlink "ruptime" => "rwho"
+    man1.install "man/ruptime.1"
+    #man8.install "man/ruptimed.8"
+  end
+
+  test do
+    system "false"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
will work on that, promised

Reply:
better now?